std/js/html/CanvasElement.hx

```diff
@@ -21,7 +21,6 @@
  */
 
 // This file is generated from mozilla\HTMLCanvasElement.webidl. Do not edit!
```

Comment:
^

Reply:
I will regen html externs after this pr, there is already changes from other contributors as i remember, need to hack more things.